src/components/tx/TxData.js

```diff
@@ -844,6 +844,20 @@ class TxData extends React.Component {
       return tokenData && tokenData.meta && tokenData.meta.nft;
     };
 
+    const isBlueprint = () => {
+      return this.props.transaction.version === 6; // hathorLib.constants.ON_CHAIN_BLUEPRINTS_VERSION
+    };
+
+    /**
+     * FIXME: The "On-Chain Blueprint" transaction type should be included on the lib util.
+     */
+    const getTxType = () => {
+      if (this.props.transaction.version === 6) {
```

Comment:
suggestion(non-blocking): 

```suggestion
      if (isBlueprint()) {
```

Reply:
Done